### ISSUE TITLE
Parsons problems: Accessibility

### ISSUE THREAD

**Issue by timlowe** [maths/moodle-qtype_stack#1627]
I asked one of our accessibility testers to take a look at a parson's problem question.

Unforunately
a) It was not readable by a screenreader
b) It did not seem to support keyboard-only navigation/manipulation.

I suspect this might be due to the underlying library - but thought I should raise the issue.

**Comment by EJMFarrow** [comment]
I've been doing some experimenting with possible accessible alternatives. Questions would display as normal but there would be an option to toggle to a more keyboard and screen-reader friendly version. Early days and posting here for comment.

(Actual code is an AI dog's breakfast, particularly with supplying of aria strings - that needs hugely reworked to handle translations. We'd also need to fix the display of MathJax in dropdowns for full compatibility.)

<img width="843" height="712" alt="Image" src="https://github.com/user-attachments/assets/c6790eab-c501-4938-afbc-25ab3314c5d9" />
<img width="746" height="614" alt="Image" src="https://github.com/user-attachments/assets/1d1604b4-d8d4-4a1d-a638-96489d988d60" />
